Content.Server/Chemistry/Components/InjectorComponent.cs

```diff
@@ -64,5 +81,7 @@ public InjectorToggleMode ToggleState
                 Dirty();
             }
         }
+
+        [ViewVariables] public BoundUserInterface? UserInterface => Owner.GetUIOrNull(TransferAmountUiKey.Key);
```

Comment:
Is this needed? Doesn't look like it's used.

Reply:
RIght. This was used in the custom transfer amount that was stripped out, thanks.